docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -1,37 +1,32 @@
 # Configuration
-To extract the configuration metadata from an application/library the _component extractor_ will be used. It extracts the list of all the config interfaces defined in a library or application. The output is a json file `component.config.metadata.json` containing an array of all the components configurations (app configs, pages, components).
-It will also generate the component classes and types metadata from an Otter library/application, outputting them in a json file `component.class.metadata.json`.
+The component extractor is used to extract the configuration metadata from an application or a library. It extracts the list of all the config interfaces defined
+in a library or an application. The output is a JSON file `component.config.metadata.json` containing an array of all the component configurations (app configurations, pages, components).
+It also generates the metadata for component classes and types from an Otter library/application, outputting them in a JSON file `component.class.metadata.json`.
 
-Note that the component.class.metadata.json is used at the moment only in rules engine scope, but in the future will be a requirement for the next steps of the Adobe SPA Editor.
 ## How to use it
 
-Since the v3 of Otter, you should use the component-extraction builder directly in your angular.json
+The Component Extractor is accessible via this NgCLI builder: `@o3r/components:extractor`.
 
-First thing to do is to define your given filenames for the configuration in the _package.json_ of the library/app where you run the extractor.
-When running in a library it will use this configuration as the names for the metadata files.
-When running the extractor in an application, it will search for these filenames in each node_module (package.json file) of each library,
-in order to concat the metadata from the file with other libraries metadata and app metadata.
+First, define your  filenames for the configuration in the `package.json` of the library/application where you run the extractor.
+When the extractor is run in a library, it uses this configuration to name the metadata files.
```

Comment:
comment from @cpaulve-1A:
which names are we talking about? The input or output? Names and filenames is a bit generic

https://github.com/AmadeusITGroup/otter/pull/1583/files#r1555823605

Reply:
Technically, let's say it is used by the extractor to find the metadata files of a dependency to compile its content into the whole package metadata and as the path where to save the metadata file when the command is generating this file.
Not sure how to phrase it though.

Reply:
It is less about application vs library than package to build vs dependencies

Reply:
When the extractor is run in a project, it will use this configuration to name of the output file of the extraction. In case you want to compile the metadata of the project with its dependencies, the extractor will need to find the metadata for each dependency.
In that case, the extractor will search for this file names...